src/store/store.js

```diff
@@ -48,7 +48,10 @@ export const store = new Vuex.Store({
         "titration":{
             "labManual": "https://stem-ap.polyu.edu.hk/remotelab/assets/files/titration.pdf",
             "setupPhoto": "https://stem-ap.polyu.edu.hk/remotelab/assets/images/titration.png"
-        }
+        },
+        "ultrasound":{
```

Comment:
if labManual and setupPhoto are undefined, it may causes problem now. You can put a empty string to them first